transformer/redshift_types.go

```diff
@@ -22,7 +24,7 @@ import (
 // for time transformers. Transform generators allow the user to define how
 // the transformer should parse a inbound property.
 
-// Note: all times are in PST.
+// All times are in PST.
```

Comment:
this isn't golint compatible, I think you guys should start making these comments golint compatible - godoc is so good.